apps/src/p5lab/AnimationTab/NewListItem.jsx

```diff
@@ -57,31 +55,31 @@ const styles = {
     display: 'block',
     paddingTop: '100%'
   },
-  dottedBorder: {
+  border: {
     position: 'absolute',
     top: 0,
     bottom: 0,
     left: 0,
     right: 0,
     borderRadius: 10,
-    border: 'dashed 2px ' + color.light_gray,
+    backgroundColor: color.bootstrap_button_blue,
```

Comment:
Hmm yeah I'm curious what Kylie says. I used a color picker online and got https://imagecolorpicker.com/color-code/2596be as the color from the spec, but that's not in our codebase it appears (although it's hard to know for sure). It looks like maybe `cyan` in our `shared/css/color.scss` file might be closer, but what you found with `bootstrap_button_blue` does seem closer to the spec? See if Kylie has an opinion about a new color because to Erin's point, we might want to use a color more common in our codebase.

Whenever I need to update color, I get curious about contrast for accessibility (honestly not sure if most of the things on this panel are color accessibility). This is the tool I use https://webaim.org/resources/contrastchecker/ and it looks like the Web Accessibility Initiative has these guidelines for focus state: https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-enhanced (hover is not necessarily focus either––this is the best I could find). When I put the colors into the webaim.org tool, I get this
<img width="723" alt="image" src="https://user-images.githubusercontent.com/9142121/178007297-d3a4f2a4-63ce-4bfc-82b7-6e279b9f6699.png">
which doesn't satisfy the ratio requirement in the w3.org link (4 ½ : 1). BUT again, I'm not sure how much it matters here, especially since other parts of this page are probably not meeting contrast standards. I'd defer to @epeach on this as she has a much more holistic view of what to prioritize and not for accessibility stuff.

Reply:
Thanks so much Meg - really awesome resource. Here's a short video showing the contrast between Button.Blue and color with hover:

https://user-images.githubusercontent.com/107423305/178015823-dadf4569-0499-4cdd-b0f3-6fbf0cdfd828.mp4